core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+**[R20]**	The (commercial) counterparties MUST agree on the business process rules which are represented in the business workflows and worksteps in the BPI. 
+
+**[R21]** 	The (commercial) counterparties MUST validate the correctness of a (commercial) state object based on a commercial state change against the transaction business logic in the applicable BPI workflow and workstep.
+
+**[R22]** 	The (commercial) counterparties MUST generate a proof of correctness of a (commercial) state object based on a commercial state change that can be validated against the BPI transaction business logic.
+
+**[R23]** 	Any new (commercial) state between counterparties MUST be recorded on the BPI between them.
+
+**[R24]** 	Any counterparty having received a proof of correctness of a (commercial) state change MUST be able to validate that proof of correctness aginst the BPI between the counterparties.
+
+**[R25]** 	A (commercial) counterparty MUST include a verifiable proof of correctness of the (commercial) state object generated by the commercial state change in the BPI Messages between the transacting counterparties.
+
+### 2.8.2 DLTs and BPI/DLT Abstraction Layers
+
+<figure>
+  <img
+  src="./images/Baseline-Fig2-DLT-BPI-Abstractions.png"
+  >
+  <figcaption>Figure 2: DLT and BPI Abstraction Architecture</figcaption>
+</figure>
+
+To maintain modularity in the reference architecture, we introduce the concept of DLT and BPI Abstraction. A DLT or BPI Abstraction expressed through a DLT Abstraction Layer, constitutes technology applications which wrap capabilities of DLTs and BPIs such that these capabilities can be exposed to applications above the DLT or BPI Abstraction Layers in a manner that minimizes the dependency of these application on the details of a DLT and BPI – Figure 2. 
+
+The Client DLT API as an external BPI API is implementation specific and will not be discussed further.
+
+**[R26]**	DLTs used in the implementation of a BPI MUST support bilateral and multi-lateral digital representations of legal contracts.
+
+**[R27]**	A DLT or BPI Abstraction Layer used in a BPI MUST support more than one DLT instance.
+
+**[R28]** A DLT or BPI Abstraction Layer used in a BPI MUST support more than one DLT type.
+
+This approach avoids lengthy discussions about which DLT protocol to utilize for a BPI, simplifying the decision making process considerably if most common DLTs are incorporated.
+
+Note that irrespective of whether one is in a public or private DLT scenario, the settings of a protocol such as block time, consensus model, type of execution framework etc. needs to be agreed upon by operating entities in some fashion either informally such as in Ethereum or formally such as in the Trade Finance consortium Komgo. 
```

Comment:
etc.,

Reply:
informally, such as in the case of Ethereum, or form..

Reply:
"the settings of a protocol" --> "protocol settings"